go.mod

```diff
@@ -49,48 +49,48 @@ require (
 	github.com/hashicorp/raft v1.3.2-0.20210825230038-1a621031eb2b
 	github.com/hashicorp/raft-boltdb v0.0.0-20171010151810-6e5ba93211ea
 	github.com/im7mortal/kmutex v1.0.1
-	github.com/juju/ansiterm v0.0.0-20210929141451-8b71cc96ebdc
+	github.com/juju/ansiterm v1.0.0
 	github.com/juju/blobstore/v2 v2.0.0-20220204135513-4876189e56a4
 	github.com/juju/charm/v8 v8.0.0-20220509231111-ed6d505a46f4
 	github.com/juju/charmrepo/v6 v6.0.0-20220207014006-e6af52d614e4
-	github.com/juju/clock v0.0.0-20220203021603-d9deb868a28a
-	github.com/juju/cmd/v3 v3.0.0-20220203030511-039f3566372a
-	github.com/juju/collections v0.0.0-20220203020748-febd7cad8a7a
-	github.com/juju/description/v3 v3.0.0-20220705143837-6a522a3e3bbc
-	github.com/juju/errors v0.0.0-20220622220526-54a94488269b
-	github.com/juju/featureflag v0.0.0-20220207005600-a9676d92ad24
-	github.com/juju/gnuflag v0.0.0-20171113085948-2ce1bb71843d
-	github.com/juju/gojsonschema v0.0.0-20150312170016-e1ad140384f2
+	github.com/juju/clock v1.0.0
+	github.com/juju/cmd/v3 v3.0.0
+	github.com/juju/collections v1.0.0
+	github.com/juju/description/v3 v3.0.1
+	github.com/juju/errors v1.0.0
+	github.com/juju/featureflag v1.0.0
+	github.com/juju/gnuflag v1.0.0
+	github.com/juju/gojsonschema v1.0.0
 	github.com/juju/gomaasapi/v2 v2.0.0-20220215192228-f1f389204471
```

Comment:
gomaasapi is a few commits behind tip of v2 branch
and there's no tags - could we add a tag to the gomaasapi branch and then use that here

Reply:
I released a v2.0.0 with the corresponding tag.